### DISCUSSION TITLE
Request timeout without user's request

### DISCUSSION THREAD

**Discussion by Juju84** [dbeaver/dbeaver#13429] [Q&A]
Hi,

I googled as much as I could without finding an answer. So here is my issue : I can't find how to intentionnaly set a timeout limit for a request. 
The idea is really to cancel a request (without manual action) if it takes more than a xxx milliseconds. 

Have I missed something ? 

Thanks in advance for your help !

**Comment by uslss** [comment]
![image](https://user-images.githubusercontent.com/31996417/128057595-78526889-2285-447f-adca-7bfae3bf4856.png)
